api/api/spec/spec.yaml

```diff
@@ -1445,6 +1445,14 @@ components:
             minor:
               type: string
           description: "Agent OS information"
+        status_code:
+          type: integer
+          description: "Agent connection status"
```

Comment:
I chose this descriptions even though the field is an enum of disconnected reasons because when it's `0` it means the connection is active.